cmd/api/src/api/v2/saved_queries.go

```diff
@@ -178,7 +178,7 @@ func (s Resources) UpdateSavedQuery(response http.ResponseWriter, request *http.
 	} else if err := api.ReadJSONRequestPayloadLimited(&updateRequest, request); err != nil {
 		api.WriteErrorResponse(request.Context(), api.BuildErrorResponse(http.StatusBadRequest, err.Error(), request), response)
 		return
-	} else if savedQueryID, err := strconv.Atoi(rawSavedQueryID); err != nil {
+	} else if savedQueryID, err := strconv.ParseInt(rawSavedQueryID, 10, 64); err != nil {
```

Comment:
Switching to ParseInt ensures we always get a base10 int64 value, rather than an `int` value